packages/coding-standard/docs/phpstan_rules.md

```diff
@@ -2355,6 +2355,56 @@ return static function (ContainerConfigurator $containerConfigurator): void {
 
 <br>
 
+## Check Required use Interface in Constract namespace
+
+- class: [`CheckRequiredInterfaceInContractNamespaceRule`](../src/Rules/CheckRequiredInterfaceInContractNamespaceRule.php)
+
+```yaml
+# phpstan.neon
+services:
+    -
+        class: Symplify\CodingStandard\Rules\CheckRequiredInterfaceInContractNamespaceRule
+        tags: [phpstan.rules.rule]
+```
+
+```php
+<?php
+
+
+declare(strict_types=1);
+
+namespace A\Contract;
```

Comment:
The example should have a non-`Contract` namespace, e.g.

```suggestion
namespace A\Controller;
```

Reply:
implemented. Added example of allow non-interface in non-"Contract" namespace.

Reply:
implemented. Added example of allow non-interface in "Contract" namespace.